lms/djangoapps/discussion_api/api.py

```diff
@@ -104,14 +104,15 @@ def _is_user_author_or_privileged(cc_content, context):
     )
 
 
-def get_thread_list_url(request, course_key, topic_id_list=None):
+def get_thread_list_url(request, course_key, topic_id_list=None, following=False):
     """
     Returns the URL for the thread_list_url field, given a list of topic_ids
     """
     path = reverse("thread-list")
     query_list = (
         [("course_id", unicode(course_key))] +
-        [("topic_id", topic_id) for topic_id in topic_id_list or []]
+        [("topic_id", topic_id) for topic_id in topic_id_list or []] +
+        ([("following", following)] if following else [])
```

Comment:
No; the following query parameter cannot be false. I could make that a little clearer by making the default value None and checking `following is not None`.